geometry_msgs/Point position
---
uav_simulator/State state
bool success